### ISSUE TITLE
NUnit1032 - disposals in overridden methods not detected

### ISSUE THREAD

**Issue by PiotrKlecha** [nunit/nunit.analyzers#921]
Object disposal that occurs in an overriden method invoked by a base test class is not detected and a diagnostic is reported.

An simplified example that demonstrates the problem:
```cs
[TestFixture]
public class BaseTest
{
	[SetUp]
	public void SetUp()
	{
		OnSetUp();
	}

	[TearDown]
	public void TearDown()
	{
		OnTearDown();
	}

	protected virtual void OnSetUp()
	{
	}

	protected virtual void OnTearDown()
	{
	}
}

[TestFixture]
public class InheritedTest : BaseTest
{
	private IDisposable? disposable;

	protected override void OnSetUp()
	{
		base.OnSetUp();
		disposable = new ADisposable();
	}

	protected override void OnTearDown()
	{
		disposable?.Dispose();
		base.OnTearDown();
	}
}
```

**Comment by manfred-brands** [comment]
I create a PR where you can specify the additional setup and teardown methods:
```
dotnet_diagnostic.NUnit1032.additional_setup_methods = OnSetUp
dotnet_diagnostic.NUnit1032.additional_teardown_methods = OnTearDown
```

The methods must be `override` as otherwise they cannot be called by the base class.
